docs/contributing.md

```diff
@@ -144,6 +144,18 @@ If you are introducing a new feature, you need to either add tests to
 existing classes, or add a new class (if your new feature is
 significantly different from existing code).
 
+#### Old tests and new tests
+Historically, tests were written with unittest and Flask-Testing.
```

Comment:
Rather than starting by explaining how it was in the past, it might be better to explain how it's supposed to be now, and then explain why we have legacy code around. Maybe something like :

> Tests are now using [pytest](https://docs.pytest.org/). However, [...].